examples/titanic/scripts/add_train_algo_random_forest.py

```diff
@@ -49,38 +56,39 @@
         z.write(filepath, arcname=os.path.basename(filepath))
 ALGO['file'] = archive_path
 
-########################################################
-#       Load keys for dataset and objective
-########################################################
-
-assets_keys_path = os.path.join(current_directory, '../assets_keys.json')
-with open(assets_keys_path, 'r') as f:
-    assets_keys = json.load(f)
 
 ########################################################
 #         Add algo
 ########################################################
 
 print('Adding algo...')
-algo_key = client.add_algo({
-    'name': ALGO['name'],
-    'file': ALGO['file'],
-    'description': ALGO['description'],
-}, exist_ok=True)['pkhash']
+algo_key = client.add_algo(ALGO, exist_ok=True)['pkhash']
 
 ########################################################
-#         Add traintuple
+#         Add traintuples
 ########################################################
 
+# Retrieve dataset and objective
+data_manager_keys = [d['key'] for d in client.list_dataset() if DATASET_NAME in d['name']]
+objective_key = [d['key'] for d in client.list_objective() if OBJECTIVE_NAME in 
+                 d['name']][0]
+traintuple_key = None
+traintuple_keys = []
+
 print('Registering traintuple...')
-traintuple = client.add_traintuple({
-    'algo_key': algo_key,
-    'objective_key': assets_keys['objective_key'],
-    'data_manager_key': assets_keys['dataset_key'],
-    'train_data_sample_keys': assets_keys['train_data_sample_keys']
-}, exist_ok=True)
-traintuple_key = traintuple.get('key') or traintuple.get('pkhash')
-assert traintuple_key, 'Missing traintuple key'
+for data_manager_key in data_manager_keys:
+    data_keys = client.get_dataset(data_manager_key)['trainDataSampleKeys']
+    traintuple = client.add_traintuple({
+        'algo_key': algo_key,
+        'objective_key': objective_key,
+        'data_manager_key': data_manager_key,
+        'train_data_sample_keys': data_keys,
+        'in_models_keys': [traintuple_key]
+    }, exist_ok=True)
+    traintuple_key = traintuple.get('key') or traintuple.get('pkhash')
+    print(traintuple_key)
```

Comment:
was it a debug print here or is it wanted?

Reply:
thanks @inalgnu it was indeed a debug print :-)